### PR TITLE
Adapting partial code(file name start with E) to the sonar cloud rule

### PR DESCRIPTION
**Pull request by gabrywu**

### CODE REVIEW

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/ExecutorService.java [hunk 1]
```diff
@@ -378,10 +376,10 @@ private boolean checkTenantSuitable(ProcessDefinition processDefinition) {
         List<Integer> ids = new ArrayList<>();
         processService.recurseFindSubProcessId(processDefineId, ids);
         Integer[] idArray = ids.toArray(new Integer[ids.size()]);
-        if (ids.size() > 0){
+        if (!ids.isEmpty()){
```
Comment (Technoboy-) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/ExecutorService.java]: Using CollectionUtils.isNotEmpty()

Comment (gabrywu) [reply dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/ExecutorService.java]: Why? check if null additional? unnecessary

**Comment by qiaozhanwei** [comment]
please resolve conflicting files

**Comment by gabrywu** [comment]
> 
> 
> please resolve conflicting files

done

**Pull request merged**